frontend/src/components/Shared/HeroBillboard/HeroBillboard.jsx

```
@@ -1,103 +1,154 @@
-        <VideoPlayer 
-          videoSource={content.videoUrl}
-          title={content.title}
-          onTrackingData={handleTrackingData}
-          previewImage={content.imageUrl}
-        />
+      <div className="absolute bottom-0 left-0 p-6 md:p-12 w-full md:w-2/3 z-10">
+        <div className="mb-4 flex items-center">
+          <span className="text-sm font-bold tracking-wide text-blue-400">{getCategoryTag()}</span>
+          {formatDuration() && (
+            <span className="ml-4 text-sm font-medium text-gray-300 flex items-center">
+              <svg className="w-4 h-4 mr-1" fill="none" stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)